client/cmd/dexc/main.go

```diff
@@ -68,7 +79,7 @@ func main() {
 	core.UseLoggerMaker(logMaker)
 	log = logMaker.Logger("DEXC")
 
-	clientCore, err := core.New(&core.Config{
+	clientCore, err = core.New(&core.Config{
```

Comment:
I don't see a `wallet.Unlock` loop like what is done on `(*Core).Logout` anywhere in the `Core` shutdown sequence, although maybe that's not a task we need to address in this PR.  Am I missing it somewhere?

Reply:
Yeah, if people agree to shutdown, should lock wallet & account

Reply:
What I'm also pointing out is that the normal shutdown, when the Core runner returns in https://github.com/decred/dcrdex/commit/0cc78f0d3a07065c5510b2cddc46e39ae7edbf3c#diff-710b0624363d9f050dd843a06c4efbc4R93-R96, did not seem to lock wallets either.  I feel like I must be missing it though.  Before these changes, when dexc shutdown, did it leave the wallets unlocked?

Reply:
I changed and pushed before your comment, kindly check my new commit
Thanks

Reply:
Yes, I was responding to the change.  Say `ok` is true in `PromptShutdown`, the wallets do not seem to be locked when dexc shuts down.  This would seem to be a bug prior to this PR, unless the wallet locking evades my attention.  Also it suggests that `(*Core).Run` should lock the wallets after `c.wg.Wait()`.  This would eliminate the need to do it in `PromptShutdown` and cover the cases where no prompt was needed too.